src/routes/UserRoutes.ts

```diff
@@ -29,6 +29,8 @@ export class UserRoutes {
 
 		router.post('/reset_password', getUser(TokenType.PasswordReset), this.userController.resetPassword.bind(this.userController));
 
+		router.get('/discord/:id', this.discordController.getDiscordUser.bind(this.discordController));
```

Comment:
This is sensitive info, I'd say that the route should be restricted to authorised admins.